src/zope/app/localpermission/tests.py

```diff
@@ -12,13 +12,22 @@
 #
 ##############################################################################
 """Tests for zope.localpermission.
-
-$Id$
 """
 import unittest
 import doctest
 
+from zope.testing.cleanup import CleanUp
+
+import zope.app.localpermission
+
+class TestConfiguration(CleanUp, unittest.TestCase):
+
+    def test_configuration(self):
+        from zope.configuration import xmlconfig
+        xmlconfig.file('configure.zcml', package=zope.app.localpermission)
+
 def test_suite():
     return unittest.TestSuite((
         doctest.DocTestSuite('zope.app.localpermission.permission'),
+        unittest.makeSuite(TestConfiguration),
```

Comment:
`unittest.defaultTestLoader.loadTestsFromName(__name__)`, in case somebody comes by later and adds another unit test class?